libs/template/presets/resources/icons/storefront/font-icons.ts

```diff
@@ -0,0 +1,123 @@
+import {
+  IconMapper,
+  IconSource,
+  ResourceFonts,
+} from '@spryker-oryx/experience';
+import { IconTypes } from '../icon.model';
+
+export const storefrontMaterialIcons: IconMapper = {
+  id: 'material-icons',
+  mapping: {
+    [IconTypes.Increase]: 'add', // merge with add
+    [IconTypes.Back]: 'chevron_left',
+    [IconTypes.Carrier]: 'local_shipping',
+    [IconTypes.Collapse]: 'expand_more',
+    [IconTypes.Mark]: 'expand_more', // merge with Collapse
+    [IconTypes.Dropdown]: 'expand_more', // merge with Collapse
+    [IconTypes.Close]: 'close',
+    [IconTypes.Calendar]: 'calendar_month',
+    [IconTypes.Expand]: 'expand_less',
+    [IconTypes.DropUp]: 'expand_less', // merge with expand
+    [IconTypes.InputError]: 'error', // merge with error
+    [IconTypes.Minus]: 'remove',
+    [IconTypes.Decrease]: 'remove', // merge with minus
+    [IconTypes.Printer]: 'print',
+    [IconTypes.Trash]: 'delete',
+    [IconTypes.Rocket]: 'rocket',
+    [IconTypes.Success]: 'check_circle',
+    [IconTypes.Invisible]: 'visibility_off',
+    [IconTypes.Wishlist]: 'favorite',
+    [IconTypes.Location]: 'location_on',
+    [IconTypes.NavigationArrow]: 'arrow_right_alt',
+    [IconTypes.Actions]: 'more_vert',
+    [IconTypes.User]: 'person',
+    [IconTypes.Image]: 'photo_library',
+    [IconTypes.ModeLight]: 'light_mode',
+    [IconTypes.ModeDark]: 'dark_mode',
+    [IconTypes.Ratings]: 'reviews',
+    [IconTypes.Planet]: 'language',
+    [IconTypes.Locker]: 'lock',
+    [IconTypes.Images]: 'images',
+  },
+};
```

Comment:
This is the material design icon set. Can we:
- rename it to `materialDesignIcons`
- move it to `icons/material-design.icons.ts` (or similar name)

The intend of this definition is that it is not storefront related.